install-git-hooks.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+## Pre-commit
+if ! cargo make --version 2&>/dev/null; then
+    echo "============================="
+    echo "=== Installing cargo-make ==="
+    echo "=============================\n"
+    cargo install cargo-make
+fi
+
+echo "============================"
+echo "=== Copy pre-commit hook ==="
+echo "============================\n"
+cp .pre-commit.sh .git/hooks/pre-commit
+chmod +x .git/hooks/pre-commit
+
+## Commit-msg
+if stat .git/hooks/commit-msg 2&>/dev/null; then
+    echo "========================================"
+    echo "=== Installing sailr commit-msg hook ==="
+    echo "========================================\n"
+    # sailr requires has `jq` to be installed
+    curl -o- https://raw.githubusercontent.com/craicoverflow/sailr/master/scripts/install.sh | bash
```

Comment:
Is there a way to do this where we don't trust the maintainers of this repo? They could theoretically replace this with a malicious script right?

Reply:
I forked the repo so we should be good.

Reply:
I was planning on forking the repo anyway because the text formatting is broken (at least on my machine).